### ISSUE TITLE
Complete COLRv1 and bitmap color rendering with tiny-kia

### ISSUE THREAD

**Issue by xorgy** [linebender/parley#105]
Currently, we have an example of rendering with tiny-skia, but it does not support COLRv1 (because tiny-skia is missing certain gradient types) and it does not support bitmap color glyphs.

**Comment by nicoburns** [comment]
Some notes:
- The bitmap glyphs are (were?) a limitation of `skrifa` not having good support for extracting the bitmaps
- The COLRv1 was indeed because of tiny-skia missing gradient types. There was mention of these being implemented in some comments starting [here](https://github.com/RazrFalcon/ttf-parser/pull/145#issuecomment-2102085113), but if any work on that has been started then I don't think it is public

(@razrfalcon is gradient types in tiny-skia still something you're planning to look into at some point?)

Previously written up [here](https://github.com/RazrFalcon/tiny-skia/issues/1#issuecomment-2131286549)

**Comment by RazrFalcon** [comment]
Yes, I do plan adding Sweep gradients eventually. But no timeline in mind.

**Comment by yisibl** [comment]
Sweep gradients have already been implemented in https://github.com/linebender/tiny-skia/pull/166
